src/main/java/frc/team2412/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,133 @@
+package frc.team2412.robot.subsystems;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LimelightSubsystem extends SubsystemBase {
+
+	// POTENTIAL CONSTANTS
+	public static final double GOAL_DISTANCE_FROM_NOTE = 20.0;
+
+	// MEMBERS
+
+	final NetworkTable networkTable;
+
+	String currentPoseString;
+	String targetPoseString;
+
+	// network tables
+
+	// CONSTRUCTOR !
+	public LimelightSubsystem() {
+
+		// camera stream at http://10.24.12.21:5800
+
```

Comment:
The Limelight should use an available static IP address between .10 and .20 (ideally 10.24.12.11). .12 is taken by the Orange PI